### PR TITLE
feat(article): add legacy article

### PR DESCRIPTION
**Pull request by DaveBitter**
Added the article on legacy code.

Checks requested on:
- title
- copy
- slug (it's rather long)
- teaser image (I thought it might be fun to see the actual interview)
- images in article

### CODE REVIEW

File: data/blog/the-infinite-what-were-they-thinking-two-years-ago-cycle-in-front-end.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: 'The infinite "what were they thinking two years ago"-cycle in front-end'
```
Comment (sanderdejong88) [inline data/blog/the-infinite-what-were-they-thinking-two-years-ago-cycle-in-front-end.md]: Other suggestions:

- Who build this?!?
- What were they thinking 2 years ago?!?
- Yeah, that was written 2 years ago
- Let's refactor that

File: data/blog/the-infinite-what-were-they-thinking-two-years-ago-cycle-in-front-end.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Naturally, this can be countered with well-written documentation, knowledge transfers and similar solutions, but you will never fully understand how it works as the original developer did. There is an acronym that we like to use called K.I.S.S. (Keep It Simple Stupid). Although this may seem overly obvious, this credo is not followed in a lot of situations. It's actually quite hard to keep your solution simple and easy to understand. The extra time invested in this will be worth it in the long run as developers not using an existing solution because it's they can't grasp what it offers or how it works is such a shame. A new solution, maybe equally as complex, is written and the cycle continues.
+
+## Front-end is **in its infancy**
```
Comment (sanderdejong88) [inline data/blog/the-infinite-what-were-they-thinking-two-years-ago-cycle-in-front-end.md]: This strong in the title isn't really noticeable. Wanna update the styling for <strong> in headings?

Comment (DaveBitter) [reply data/blog/the-infinite-what-were-they-thinking-two-years-ago-cycle-in-front-end.md]: Hmmm, not sure why that is there. Probably some notion weirdness, will remove

File: data/blog/the-infinite-what-were-they-thinking-two-years-ago-cycle-in-front-end.md (hunk 1 above)
Comment (DaveBitter) [reply data/blog/the-infinite-what-were-they-thinking-two-years-ago-cycle-in-front-end.md]: Updated to _The infinite legacy cycle in front-end_

**Pull request merged**